### PR TITLE
Create normative_content.rst

### PR DESCRIPTION
**Pull request by amy-silcock**
Adding in standalone normative content page as per: https://docs.google.com/document/d/1Y_PU_Ot4xBBA8rcMGbqqKeZWeWX9SLknVZD4nW2HKds/edit#

Then need to link to this page from guidance update process doc in guidance repo.

### CODE REVIEW

**Review by akmiller01 — APPROVED** [review]

File: normative_content.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+IATI Normative and Non-normative content
+========================================
+
+**Normative:** `Normative content <https://en.wikipedia.org/wiki/Normative#Standards_documents>`__ is the prescriptive part of a standard. It sets the rules to be followed in order to be evaluated as compliant with the standard, and from which no deviation is permitted. 
+
+The following content is considered normative:
+
+- `IATI SSOT <https://github.com/IATI/IATI-Standard-SSOT>`__
+
+  - `IATI-Codelists <https://github.com/IATI/IATI-Codelists/tree/7dc8ae401dfdf6970130eeabc08974b11ff9140b>`__
```
Comment (nosvalds) [inline normative_content.rst:10]: Did you intend to include the /tree/{commit_sha} here? If you just want to link to the general IATI-Codelists repo we should shorten the URL. Adding the /tree/{commit_sha} will lock the link into that specific commit in history.
```suggestion
  - `IATI-Codelists <https://github.com/IATI/IATI-Codelists>`__
```

File: normative_content.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- `IATI-Extra-Documentation <https://github.com/IATI/IATI-Extra-Documentation>`__
+
+- `IATI Guidance <https://github.com/IATI/IATI-Guidance>`__
+
+- `IATI-Developer-Documentation <https://github.com/IATI/IATI-Developer-Documentation>`__
+
+- `IATI-Upgrades <https://github.com/IATI/IATI-Upgrades>`__
+
+
+In addition to the non-normative content listed above that is related to the standard and managed on Github, IATI also provides `how to publish guidance <https://iatistandard.org/en/guidance/>`__ (managed on Wagtail) which also falls under this category.
```
Comment (nosvalds) [inline normative_content.rst:29]: remove trailing `/` in URL for consistency
```suggestion
In addition to the non-normative content listed above that is related to the standard and managed on Github, IATI also provides `how to publish guidance <https://iatistandard.org/en/guidance>`__ (managed on Wagtail) which also falls under this category.
```

File: normative_content.rst [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+**Normative:** `Normative content <https://en.wikipedia.org/wiki/Normative#Standards_documents>`__ is the prescriptive part of a standard. It sets the rules to be followed in order to be evaluated as compliant with the standard, and from which no deviation is permitted. 
+
+The following content is considered normative:
+
+- `IATI SSOT <https://github.com/IATI/IATI-Standard-SSOT>`__
+
+  - `IATI-Codelists <https://github.com/IATI/IATI-Codelists/tree/7dc8ae401dfdf6970130eeabc08974b11ff9140b>`__
+  
+  - `IATI-Rulesets <https://github.com/IATI/IATI-Rulesets/tree/d49b82b821e7ed23d62da1a6767d9dd7cdd310b6>`__
```
Comment (nosvalds) [inline normative_content.rst:12]:
```suggestion
  - `IATI-Rulesets <https://github.com/IATI/IATI-Rulesets>`__
```

File: normative_content.rst [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  
+  - `IATI-Schemas <https://github.com/IATI/IATI-Schemas/tree/9119bb5fa7d042d81f6cbac9a20970350b723272>`__
```
Comment (nosvalds) [inline normative_content.rst:14]:
```suggestion
  - `IATI-Schemas <https://github.com/IATI/IATI-Schemas>`__
```

**Review by nosvalds — CHANGES_REQUESTED** [review]
@amy-silcock can you change the filename to normative-content.rst to follow the naming pattern of the other pages?

File: normative_content.rst (hunk 1 above)
Comment (amy-silcock) [reply normative_content.rst:10]: Good spot! I'd just copied and pasted from the google doc. I'll change them

**Review by amy-silcock — COMMENTED** [review]
Thanks Nik these all are good!

**Review by nosvalds — APPROVED** [review]
Looks great!

**Review by PetyaKangalova — APPROVED** [review]
Checked links and content looks good, so approving!

**Pull request merged**